addon/components/custom-subsidy-form-fields/plan-living-together-table/edit.hbs

```diff
@@ -0,0 +1,510 @@
+<div class="au-u-margin-bottom">
+  <h3 class="au-heading-3 au-u-medium">Hieronder vind je alle acties die gesubsidieerd worden door het Plan Samenleven.</h3>
+  <ul class="au-u-regular">
+    <li class="au-c-help-text">&#9679; Geef in de tweede kolom aan wat het huidige bereik is voor deze actie (zonder middelen van het Plan Samenleven). Dit moet enkel ingevuld worden als jullie ook bijkomende middelen hiervoor inzetten.</li>
+    <li class="au-c-help-text">&#9679; Geef in de derde kolom aan welke bijkomende doelstellingen jullie bestuur wenst te realiseren voor het Plan Samenleven. Enkel deze bijkomende acties komen in aanmerking voor subsidiering door de Vlaamse overheid.</li>
+    <li class="au-c-help-text">&#9679; In de vierde kolom ziet u het bedrag dat in aanmerking komt voor subsidiering door de Vlaamse overheid. Er wordt verwacht dat het bestuur voor deze acties evenveel financiert.</li>
+    <li class="au-c-help-text">&#9679; Geef in de vierde kolom aan de hand van cijfers aan wat de prioriteit is per actie waarop het bestuur bijkomend inzet. Wanneer er niet voldoende middelen zijn, zal de Vlaamse overheid aan de hand van deze prioritering bepalen naar welke acties de subsidies gaan. Wanneer jullie bestuur intekent voor bv. 5 acties, dan moeten de nummer 1 t.e.m. 5 verdeeld worden over deze acties.</li>
+  </ul>
+</div>
+
+<AuLabel class="{{if this.errors " au-c-label--error"}}" for={{this.inputFor}}>
```

Comment:
`@error` should be used to have proper styling in the newer appuniversum versions.
```suggestion
<AuLabel @error={{if this.errors true false}} for={{this.inputFor}}>
```